### DISCUSSION TITLE
Problem after reboot

### DISCUSSION THREAD

**Discussion by Rhoenit** [telekom-security/tpotce#846] [Q&A]
Hello,

after the daily reboot sometimes all containers stay down. If I reboot manualy the system requires a manual fsck. Now the T-Pot works for some days until it happens again. 

It doesn´t mater if it is a VM or local machine. 

Does anyone have same problems or a fix?

**Comment by t3chn0m4g3** [comment]
Unfortunately I cannot reproduce this and `reboot` usually ensures on the OS side that all services are shutdown.

In `/etc/crontab` we also ensure that T-Pot services are shut down graciously, you can however modify this.
```
systemctl stop tpot && docker stop $(docker ps -aq) || docker rm $(docker ps -aq) || reboot
```
Are you using SSDs or HDDs?

**Reply by Rhoenit** [reply]
I´m using SSDs.

**Reply by t3chn0m4g3** [reply]
I recommend to monitor the [health](https://linuxhandbook.com/check-ssd-health/) of the SSD.

Unfortunately there is not much I can do, there are so many factors playing into this, i.e. chipset, mainboard, RAM, drivers, etc. You can look into [this ](http://forums.debian.net/viewtopic.php?t=133602) and default `fsck` to yes.

**Reply by Rhoenit** [reply]
I couldn´t find any problem with the SSD. 

The system boots but the containers doens´t work and I can´t connect through network. So default fsck to yes wont help that much. 

Are there any logs where I can maybe find something?

Which hardware and VM Options do you use?

**Reply by t3chn0m4g3** [reply]
Except for AWS and GCP we have them all. ESXi, OTC, VirtualBox, KVM, Intel NUC, Commodity Hardware, etc. As long minimum requirements are met it works flawlessly.
However I made some changes a couple of weeks ago and adjusted the specs for the Elastic containers. Run the update.sh script to retrieve the changes.

**Reply by Rhoenit** [reply]
![initramfs](https://user-images.githubusercontent.com/78480832/121683502-c654ad80-cabd-11eb-925e-73bff7cde70d.PNG)

The next reboot after the update :(

Does this path help you to find something?
